index.html

```diff
@@ -17,11 +17,26 @@
 </head>
 <body>
   <header>
-    <div class="container">
-    </div>
+    <p id="header-p" class="header-P">Code Journal</p>
   </header>
   <main class="container">
-    <div data-view="entry-form"></div>
+    <h1>New Entry</h1>
+    <div class="container">
+      <input type="image" src="images/placeholder-image-square.jpg" class="url-pic">
+    </div>
+    <div class="container">
+      <div data-view="entry-form">
+        <form id="form">
+          <label for="title">Title</label>
+          <input type="text" id="title" name="title" required>
+          <label for="photoURL">Photo URL</label>
+          <input type="text" id="photo-url" name="photo-url" required>
+          <label for="notes">Notes</label>
+          <input type="text" id="notes" name="notes" required>
+          <input type="submit" id="save-button" value="SAVE">
+        </form>
```

Comment:
You're not utilizing `row`s and `column`s. You should be utilizing `div` elements with the layout classes we provided for you to break these up into `column`s